### PR TITLE
Add super.toString() to Cognito events

### PR DESCRIPTION
**Pull request by NikolayMetchev**
*Issue #, if available:*
https://github.com/aws/aws-lambda-java-libs/issues/433

*Description of changes:*
Add super.toString() to cognito generated classes

*Target (OCI, Managed Runtime, both):*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/CognitoUserPoolPostAuthenticationEvent.java [hunk 1]
```diff
@@ -12,10 +12,7 @@
  */
 package com.amazonaws.services.lambda.runtime.events;
 
-import lombok.Builder;
-import lombok.Data;
-import lombok.EqualsAndHashCode;
-import lombok.NoArgsConstructor;
+import lombok.*;
```
Comment (msailes) [inline aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/CognitoUserPoolPostAuthenticationEvent.java:15]: Could you revert this change please?

File: aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/CognitoUserPoolPostConfirmationEvent.java [hunk 2]
```diff
@@ -12,10 +12,7 @@
  */
 package com.amazonaws.services.lambda.runtime.events;
 
-import lombok.Builder;
-import lombok.Data;
-import lombok.EqualsAndHashCode;
-import lombok.NoArgsConstructor;
+import lombok.*;
```
Comment (msailes) [inline aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/CognitoUserPoolPostConfirmationEvent.java:15]: Could you revert this change please.

File: aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/CognitoUserPoolPreAuthenticationEvent.java [hunk 3]
```diff
@@ -12,10 +12,7 @@
  */
 package com.amazonaws.services.lambda.runtime.events;
 
-import lombok.Builder;
-import lombok.Data;
-import lombok.EqualsAndHashCode;
-import lombok.NoArgsConstructor;
+import lombok.*;
```
Comment (msailes) [inline aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/CognitoUserPoolPreAuthenticationEvent.java:15]: Same

**Review by smirnoal — APPROVED** [review]

**Review by anton-stepanof — APPROVED** [review]
